lib/sorbet-rails/model_plugins/active_record_assoc.rb

```diff
@@ -65,8 +68,7 @@ def populate_single_assoc_getter_setter(assoc_module_rbi, assoc_name, reflection
         !!reflection.active_record.belongs_to_required_by_default
       end
 
-    column_def = @columns_hash[reflection.foreign_key.to_s]
-    db_required_config = column_def.present? && !column_def.null
+    rails_required_config ||= [column_def&.name, reflection.name].compact.any? { |n| attribute_has_unconditional_presence_validation?(n) }
```

Comment:
can you add a comment here why we can use "reflection.name"? is it so that it checks the validation logic for both object (wizard) and column attribute (wizard_id)?